pytestqt/plugin.py

```diff
@@ -90,15 +83,45 @@ def pytest_addoption(parser):
                          'default: "{0}"'.format(default))
 
 
-@pytest.mark.hookwrapper
+@pytest.hookimpl(hookwrapper=True, tryfirst=True)
```

Comment:
I think `@pytest.hookimpl` is a pytest 2.8 thing - not sure if we should already break 2.7 compatibility as there are still some people with 2.8 regressions I think.

Maybe use `@pytest.mark.hookimpl` for now if `@pytest.hookimpl` is unavailable?